### DISCUSSION TITLE
Hide panel menus

### DISCUSSION THREAD

**Discussion by ollie-campbell** [grafana/grafana#60571] [Dashboards, Panels & Visualizations]
I've seen a few forum posts about this, but not seen a feature request that I could find.

Is there a way to hide the 'View, Edit, Share' panel menu bar?

This would be extremely useful for touch screen operation or where button panels are used. Not to mention removing any options for anonymous users.

Thanks

EDIT: This thread seems quite useful:
https://community.grafana.com/t/disable-view-share-and-more-menu-from-graph/26572

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
